### DISCUSSION TITLE
ArgoCD support for 1.29

### DISCUSSION THREAD

**Discussion by mjnovice** [argoproj/argo-cd#17813] [General]
Hey team, wanted to understand are there plans to release argo support for 1.29 version. Thanks.

**Comment by avosepp** [comment]
I'm running on 1.29.2 with Argo for a few weeks now. Everything seems fine. Is there a component that isn't supported? I'm not aware of a support issue with 1.29

**Comment by mjnovice** [comment]
Hey @avo-sepp the redis HA seems problematic on AKS.

**Comment by avosepp** [comment]
What problems have you been having with it?

**Comment by mjnovice** [comment]
The readiness/liveness probe keeps failing with timeouts.


 Apr 12 20:15:13 aks-pool0-18813482-vmss000002 kubelet[2923]: I0412
20:15:13.029954    2923 kubelet.go:2519] "SyncLoop (probe)"
probe="readiness" status="" pod="argocd/argocd-redis-ha-server-0"
 Apr 12 20:31:07 aks-pool0-18813482-vmss000002 kubelet[2923]: I0412
20:31:07.397701    2923 prober.go:107] "Probe failed" probeType="Readiness"
pod="argocd/argocd-redis-ha-server-0"
podUID="a3ae132f-07d3-4d2c-bdf1-f385ce96d4ee" containerName="redis"
probeResult="failure" output="command \"sh -c /health/redis_readiness.sh\"
timed out"

And it takes a long time to finally sync, or sometimes it doesn't even sync.

**Comment by jarviliam** [comment]
I'm running into the same thing (not the sync issue) over here on 1.29 EKS, ArgoCD 2.10. 
It eventually recovers but it keeps repeating.
